### ISSUE TITLE
Optional fine tuning: good to have

### ISSUE THREAD

**Issue by reginleiff** [this-place/this.place#208]
Here are some key but optional small blemishes that we can fix to fine tune the game experience:

- [x] Unable to jump in the tutorial walkway + some blocks not snapped to grid (see below)

![image](https://user-images.githubusercontent.com/22815234/38841898-4255d6a6-4219-11e8-98f6-11d488e313ff.png)

- [x] Destroy eyes of AI enemy and death sound
- [x] Gravel blocks in tutorial and stage 1 cause massive FPS drop (more so then previous versions of the game)
- [x] Camera clipping in Start Screen

**Remaining from #203**
- [x] End scene
- [x] Dim blocks upon displace
- [x] Coins scale into world
- [x] Reloading into awkward positions
- [ ] S2P3>S2P4 Transition is Unnatural 

will find more by today...

**Comment by Rinder5** [comment]
when you say camera clipping in start screen, you mean when the auto-rotate goes into the building? I turn off zoom while paused, but i can enable it instead if preferred

**Comment by reginleiff** [comment]
Yep, it is as you say. I think we should enable it if it hides the clipping into the building.

**Comment by Rinder5** [comment]
It looks pretty bad, the zoom is pretty jagged because the rotation is so slow, so it keeps having to adjust. Also, the house has supports, so it zooms in and out quite a bit

**Comment by reginleiff** [comment]
I see, okay then we just leave it as it is since there's no other option. I'm actually ok with a static view on the character on idle, but I remember some of us didn't like that.
